cmd/drone-bazelisk-ecr/plugin.go

```diff
@@ -24,6 +24,9 @@ type plugin struct {
 	AccessKey        string `split_words:"true"`
 	SecretKey        string `split_words:"true"`
 	Bazelrc          string
+	Command          string
+	CommandArgs      string
```

Comment:
`split_words:"true"` is needed to ensure it parses the correct env var (`PLUGIN_COMMAND_ARGS` instead of `PLUGIN_COMMANDARGS`).
```suggestion
	CommandArgs      string `split_words:"true"`
```